pkg/kube/portforwarder.go

```diff
@@ -115,7 +113,7 @@ func newPortForwarder(restConfig *rest.Config, podName, ns, localAddress string,
 		[]string{fmt.Sprintf("%d:%d", localPort, podPort)},
 		stopCh,
 		readyCh,
-		output,
+		io.Discard,
```

Comment:
the `output` writer apparently needs to be threadsafe. We don't actually read the output anyways though, so just us io.Discard is easiest